libs/redux/userTerrains/api.ts

```diff
@@ -0,0 +1,15 @@
+import clientApi from '../clientApi';
+
+const UserTerrainsApi = clientApi.injectEndpoints({
+  endpoints: (build) => ({
+    getuserTerrains: build.query({
+      query: () => `/user-terrain`,
+      providesTags: ['UserTerrainsApi'],
+    }),
+  }),
+});
+
+export const { useGetuserTerrainsQuery } = UserTerrainsApi;
```

Comment:
I believe this is the last change needed. This naming convention is a bit awkward, but actually comes from [RTK Query](https://redux-toolkit.js.org/rtk-query/overview#basic-usage)

`useGetUserTerrainsQuery` instead of `useGetuserTerrainsQuery`